connect/transforms/src/main/java/org/apache/kafka/connect/transforms/TimestampConverter.java

```diff
@@ -83,12 +78,36 @@
     private static final String TYPE_TIMESTAMP = "Timestamp";
     private static final Set<String> VALID_TYPES = new HashSet<>(Arrays.asList(TYPE_STRING, TYPE_UNIX, TYPE_DATE, TYPE_TIME, TYPE_TIMESTAMP));
 
+    private static final String UNIX_PRECISION_MILLIS = "milliseconds";
+    private static final String UNIX_PRECISION_MICROS = "microseconds";
+    private static final String UNIX_PRECISION_NANOS = "nanoseconds";
+    private static final String UNIX_PRECISION_SECONDS = "seconds";
+    private static final Set<String> VALID_UNIX_PRECISIONS = new HashSet<>(
```

Comment:
This field is not used, we can remove it.

Reply:
Thanks, sorry about this